src/catch2/catch_totals.cpp

```diff
@@ -14,21 +14,23 @@ namespace Catch {
         diff.passed = passed - other.passed;
         diff.failed = failed - other.failed;
         diff.failedButOk = failedButOk - other.failedButOk;
+        diff.skipped = skipped - other.skipped;
         return diff;
     }
 
     Counts& Counts::operator += ( Counts const& other ) {
         passed += other.passed;
         failed += other.failed;
         failedButOk += other.failedButOk;
+        skipped += other.skipped;
         return *this;
     }
 
     std::uint64_t Counts::total() const {
-        return passed + failed + failedButOk;
+        return passed + failed + failedButOk + skipped;
```

Comment:
As part of the change to not print "skipped assertions" I've decided to keep `skipped` as part of `total()` for consistency within `Counts`, and instead added special handling for `skipped` counts in the console reporter.